rewrite-maven/src/test/java/org/openrewrite/maven/AddParentPomTest.java

```diff
@@ -0,0 +1,800 @@
+                  <version>2.7.18</version>
+                </parent>
+                <groupId>org.sample</groupId>
+                <artifactId>sample</artifactId>
+                <version>1.0.0</version>
+
+                <properties>
+                  <my-cool-prop>${junit.version}</my-cool-prop>
+                </properties>
+              </project>
+              """
+          )
+        );
+    }
+
+    @Test
+    void doesNotAddGlobalProperties() {
+        rewriteRun(
+          spec -> spec.recipe(new AddParentPom("org.springframework.boot", "spring-boot-starter-parent", "2.7.18", null, null, null)),
+          pomXml(
+            """
+              <?xml version="1.0" encoding="UTF-8"?>
+              <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+                <modelVersion>4.0.0</modelVersion>
+                <groupId>org.sample</groupId>
+                <artifactId>sample</artifactId>
+                <version>1.0.0</version>
+
+                <properties>
+                  <my-basedir>${basedir}</my-basedir>
+                  <my-project-basedir>${project.basedir}</my-project-basedir>
+                  <my-project-build-directory>${project.build.directory}</my-project-build-directory>
+                  <my-project-version>${project.version}</my-project-version>
+                  <my-env-prop>${env.GIT_HOME}</my-env-prop>
+                  <my-settings-prop>${settings.offline}</my-settings-prop>
+                </properties>
+              </project>
+              """,
+            """
+              <?xml version="1.0" encoding="UTF-8"?>
+              <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+                <modelVersion>4.0.0</modelVersion>
+                <parent>
+                  <groupId>org.springframework.boot</groupId>
+                  <artifactId>spring-boot-starter-parent</artifactId>
+                  <version>2.7.18</version>
+                </parent>
+                <groupId>org.sample</groupId>
+                <artifactId>sample</artifactId>
+                <version>1.0.0</version>
+
+                <properties>
+                  <my-basedir>${basedir}</my-basedir>
+                  <my-project-basedir>${project.basedir}</my-project-basedir>
+                  <my-project-build-directory>${project.build.directory}</my-project-build-directory>
+                  <my-project-version>${project.version}</my-project-version>
+                  <my-env-prop>${env.GIT_HOME}</my-env-prop>
+                  <my-settings-prop>${settings.offline}</my-settings-prop>
+                </properties>
+              </project>
+              """
+          )
+        );
+    }
+
```

Comment:
These tests are not necessary I think; there's no corresponding logic that would cause these to be added, right?
```suggestion
```